libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -1,58 +1,23 @@
-import PauseCircleFilledIcon from '@mui/icons-material/PauseCircleFilled';
-import PlayCircleFilledIcon from '@mui/icons-material/PlayCircleFilled';
-import { useEffect, useState } from 'react';
-import styles from './MediaPlayer.module.scss';
+import { Box } from '@mui/material';
+import { useEffect, useRef } from 'react';
 
 export interface MediaPlayerProps {
     audioUrl: string;
     listenMessage?: string;
 }
 
-const DEFAULT_LISTEN_MESSAGE = 'Listen Live!';
-
-export function MediaPlayer({ audioUrl, listenMessage }: MediaPlayerProps) {
-    const [audio, setAudio] = useState<HTMLAudioElement | null>(null);
-
-    const [canPlayThrough, setCanPlayThrough] = useState(false);
-
-    const safePlay = () => {
-        if (!canPlayThrough || !audio) {
-            return;
-        }
-
-        audio.play();
-    };
-
-    //  const toggle: MouseEventHandler<HTMLButtonElement> = (): void => setPlaying(!playing);
+export function MediaPlayer({ audioUrl }: MediaPlayerProps) {
+    const audioRef = useRef<HTMLAudioElement | null>(null);
 
     useEffect(() => {
-        const audioElement = new Audio();
-        audioElement.src = audioUrl;
-        audioElement.preload = 'auto';
-        audioElement.addEventListener('canplaythrough', () => {
-            setCanPlayThrough(true);
-        });
-        setAudio(audioElement);
+        if (audioRef.current !== null) {
+            audioRef.current.src = audioUrl;
+        }
     }, [audioUrl]);
 
     return (
-        <div className={styles['container']}>
-            {audio && (
-                <>
-                    <PlayCircleFilledIcon
-                        className={styles['media-controls']}
-                        onClick={() => safePlay()}
-                    >
-                        Play
-                    </PlayCircleFilledIcon>
-                    <PauseCircleFilledIcon
-                        className={styles['media-controls']}
-                        onClick={() => audio.pause()}
-                    >
-                        Pause
-                    </PauseCircleFilledIcon>
-                </>
-            )}
-        </div>
+        <Box>
+            <audio ref={audioRef} controls data-testid="audio-player"></audio>
```

Comment:
I found a CSS stylesheet for the audio and video player here: https://chromium.googlesource.com/chromium/blink/+/72fef91ac1ef679207f51def8133b336a6f6588f/Source/core/css/mediaControls.css?autodive=0%2F%2F%2F